Packs/ArcherRSA/Playbooks/playbook-Archer_initiate_incident.yml

```diff
@@ -2,8 +2,10 @@ id: Archer initiate incident
 version: -1
 name: Archer initiate incident
 fromversion: "5.0.0"
+deprecated: true
 starttaskid: "0"
-description: "initiate Archer incident"
+description: "Deprecated. Use the `archer-get-file` command directly instead. 
```

Comment:
How get-file replacing "initiate Archer incident"?

Reply:
oh, the only task in the playbook is to get a file?

Reply:
Yes. The Playbook contains only one task with the deprecated command `archer-get-file`.